packages/frontend-v2/pages/index.tsx

```diff
@@ -1,22 +1,108 @@
+import "react-toastify/dist/ReactToastify.min.css";
```

Comment:
I also see this in the documentation. Do we have to do this in every file if we were to use toasts?

Reply:
also ^ should it be here, or in _app.tsx?

Reply:
good catch this should go in app.tsx! 

but yeap we'll need to add the styles to all our pages that use toasts... if we were using CRA then we wouldn't need to do this but with Nextjs UI libraries often find it easier to bundle their styles and have them imported since Next doesn't allow components styles to be injected in the document for any static/server side pages using `style-loader`. Dealt with this issue a ton on my co-op lmaoo.